gallery/tutorials/lecture_feature_demo.py

```diff
@@ -106,44 +94,25 @@
 v = Volume.load(os.path.join(DATA_DIR, "clean70SRibosome_vol_65p.mrc"))
 
 # More interesting data requires downloading locally.
-# v = Volume.load("EMD-2660/map/emd_2660.map")
+# v = Volume.load("path/to/EMD-2660/map/emd_2660.map")
 
 # Downsample the volume to a desired resolution
 img_size = 64
-# Volume.downsample() Returns a new Volume instance.
-#   We will use this lower resolution volume later.
+# Volume.downsample() returns a new Volume instance.
+#   We will use this lower resolution volume later, calling it `v2`.
 v2 = v.downsample(img_size)
 L = v2.resolution
 
 # %%
-# Contour Plot of Data
-# --------------------
+# Plot Data
+# ---------
 
 # Alternatively, for quick sanity checking purposes we can view as a contour plot.
 #   We'll use three orthographic projections, one per axis.
 for axis in range(3):
-    plt.contourf(np.arange(L), np.arange(L), np.sum(v2[0], axis=axis), cmap="gray")
+    plt.imshow(np.sum(v2[0], axis=axis), cmap="gray")
```

Comment:
Maybe this should have been caught earlier, but this loop doesn't work as expected in the gallery. Only one image is shown.

Reply:
Right, I think this was from when it was a jupyter notebook, but probably never worked in the gallery port. I changed it to a (1,3) subplot. Good catch, thanks.